### ISSUE TITLE
Handle Snake_Case: Treat Underscore (and Hyphen) as white space

### ISSUE THREAD

**Issue by crjeder** [drahnr/cargo-spellcheck#332]
Names for e. g. variables could be written in snake case. For the purpose of spellchecking Hyphen ("-") and Underscore ("_") should be treated as word separator.

```rust
set_mode();
```
is not an accepted word, but `set` and `mode`would be.

Interpreting snake case correctly would reduce the amount of false positives.

**Comment by crjeder** [comment]
I don't know what the problem was, but of course, snake_case works!

**Comment by crjeder** [comment]
That's the error message I am getting:
```text
    |
 69 |  Set channel and gain with the set_mode() function.
    |                                ^^^^^^^^
    | - stomodeum
    |
    |   Possible spelling mistake found.
```

**Comment by crjeder** [comment]
Did a bit more testing. Seems this only happens in documentation comments ("///")

**Comment by crjeder** [comment]
code line is:
```rust
/// Set channel and gain with the set_mode() function.
```
repository: [my hx711 driver](https://github.com/crjeder/hx711_spi/)

**Comment by drahnr** [comment]
Did you consider using back ticks for function calls? I think that'd be the appropriate common Mark context to use that is not spell checked.

**Comment by drahnr** [comment]
Also see #208 #329 which are taking a more holistic approach to the problem you outlined.

**Comment by crjeder** [comment]
jep, the workaround with the back ticks works for me.
Thx!
